.circleci/config.yml

```diff
@@ -35,6 +46,20 @@ jobs:
             - v1-{{ .Branch }}
           paths:
             - /caches/app.tar
+      - run:
+          name: Authenticate with non-production cluster
+          command: |
+            echo -n ${NON_PROD_CA_CERT} | base64 -d > ./ca.crt
+            #echo ${NON_PROD_CA_CERT} > ./encoded-ca.crt
+            #base64 -d ./encoded-ca.crt > ./ca.crt
```

Comment:
need to remove these comments

Reply:
The commented code does exactly the same as the new line - 'echo -n ${NON_PROD_CA_CERT} | base64 -d > ./ca.crt'. However, the new line is a shorter version.

Reply:
@tariqmoj Removed